### ISSUE TITLE
[ERR] NgxBootstrapIconsModule is already loaded. Import it in the AppModule only

### ISSUE THREAD

**Issue by plungarini-mayp** [avmaisak/ngx-bootstrap-icons#253]
Hello there.
I just updated NGXBootstrapIcons.

First thing I should say is that as of the lib was built, it was wery difficult to correctly mantain all the imports.
Now by changing to .pick() to .forRoot() it messed up the entire project I was working on.

I followed the instructions and I imported the lib in AppModule with .forRoot(allIcons) and I imported the lib without .forRoot() in the other modules where it needs the component.

Unfortunately I receive this error:
![image](https://user-images.githubusercontent.com/56771393/107563705-b0071580-6be1-11eb-9656-2f7676b41521.png)

This is where I put the library:
![image](https://user-images.githubusercontent.com/56771393/107563846-d75de280-6be1-11eb-8d64-af4cd4931648.png)

As you can see it should be fine, as the new guide described it.
I don't know why the behaviour is like this.

**Comment by avmaisak** [comment]
Hello @plungarini-mayp!

I checked this issue and updated demo app.
And in my cases it works. Please, see example project in this repo.

My steps:

**Added in root module**

![image](https://user-images.githubusercontent.com/1219921/107607587-b67da780-6c4a-11eb-9551-3ce1a2341161.png)

**Create feature module**

![image](https://user-images.githubusercontent.com/1219921/107607623-cbf2d180-6c4a-11eb-8574-68c5b3929dcc.png)

**In feature module i added needed icons**

![image](https://user-images.githubusercontent.com/1219921/107607655-e167fb80-6c4a-11eb-859b-a9a188e1b92b.png)

And it works fine, without any warnings

![image](https://user-images.githubusercontent.com/1219921/107607719-0fe5d680-6c4b-11eb-8d3f-f2feacad5e3b.png)


@plungarini-mayp could you provide source code, for more detailed research of your problem?

**Comment by avmaisak** [comment]
@plungarini-mayp look at these source files.

https://github.com/avmaisak/ngx-bootstrap-icons/tree/develop/src/app

i updated demo page, see

https://avmaisak.github.io/ngx-bootstrap-icons/

**Comment by plungarini-mayp** [comment]
I'll send you a demo soon. But I have a question. What if I want the NGXBICONS Module in different modules? Before this update I had a shared module with .pick(allIcons) and I exported it. But this gives me the same issue

**Comment by avmaisak** [comment]
@plungarini-mayp as solution, i may apply method forRoot(..) in appmodule, and then, in Shared module, just import module without method forRoot

**Comment by plungarini-mayp** [comment]
That's exactly what I did but it doesn't solve this unfortunately

**Comment by avmaisak** [comment]
@plungarini-mayp 
ok, i will try to reproduce this cases.

**Comment by tayambamwanza** [comment]
@avmaisak   I also received this error, I'll try and see what I can do on my side

**Comment by tayambamwanza** [comment]
@avmaisak @plungarini-mayp  I knew I'd seen this error message before!

The problem exists here:
![Screenshot from 2021-02-22 19-44-56](https://user-images.githubusercontent.com/11971384/108748362-2e9b7580-7547-11eb-8a45-8b435d38fbe6.png)

This is from the old Angular 4 docs but it was removed because they stopped recommending a CoreModule (Although you can still have one).

[Angular 4 Core Module Prevent Reimport of Core Module](https://v4.angular.io/guide/ngmodule#prevent-reimport-of-the-coremodule)

![Screenshot from 2021-02-22 19-48-48](https://user-images.githubusercontent.com/11971384/108748651-80440000-7547-11eb-8869-9fc7c530af81.png)

That concept is to prevent modules from being imported in more than one place, specifically the app module.

Remove:

```ts
@Optional() @SkipSelf() parentModule: NgxBootstrapIconsModule,
```
and

```ts
if (parentModule) throw new Error('NgxBootstrapIconsModule is already loaded. Import it in the AppModule only');
```
The problem will go away.

**Comment by plungarini-mayp** [comment]
You're amazing mate. Thanks a lot! You're a goddamn monster! Peace ✌🏼

**Comment by tayambamwanza** [comment]
@plungarini-mayp Haha don't thank me man, just helping out @avmaisak  is the real one here, he put this all together, thanks anyway, cheers.

**Comment by avmaisak** [comment]
@plungarini-mayp fixed at 1.3.6

!!! Has breaking changes !!!

* Updated to 1.4.0
* fixed #253, added config options for module,
* [breaking changes] - reverted to pick() method
